build/configs/imxrt1020-evk/loadable_elf_apps/defconfig

```diff
@@ -325,9 +325,9 @@ CONFIG_ARCH_USE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="256,256,256,256,256,256,"
-CONFIG_FLASH_PART_TYPE="kernel,none,bin,bin,bin,bin,"
-CONFIG_FLASH_PART_NAME="kernel,app,micom,micom,wifi,wifi,"
+CONFIG_FLASH_PART_SIZE="256,256,256,256,256,256,256,"
```

Comment:
How about to increase size?

Reply:
The partition size of smartfs is now 2048 (kb).